tests/plugins/pastefromword/manual/pastetable.html

```diff
@@ -0,0 +1,29 @@
+<h2>Editor</h2>
+<div id="editor"></div>
+<h2>Clipboard dump</h2>
+<label for="input">Paste clibpoard data</label>
+<input id="input" name="input" />
+<div id="result"></div>
+
+<script>
+	var input = document.getElementById( 'input' ),
+		result = document.getElementById( 'result' );
+
+	if ( !CKEDITOR.env.ie || CKEDITOR.env.version > 11 ) {
+		input.addEventListener( 'paste', function( evt ) {
```

Comment:
We should be able to get raw data directly from editor's `paste` event, see [`CKEDITOR.plugins.clipboard.dataTransfer.getData`'s 2nd parameter](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins_clipboard_dataTransfer.html#method-getData).

Reply:
Unfortunately, it still returns an empty string for IE. The input element is not a part of an editor, so it requires additional code to construct our dataTransfer object for the same results.